compose/src/main/java/com/bottlerocketstudios/compose/pullrequest/PullRequestItemCard.kt

```diff
@@ -13,68 +13,63 @@ import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.res.painterResource
 import androidx.compose.ui.res.stringResource
-import androidx.compose.ui.text.TextStyle
-import androidx.compose.ui.text.font.FontWeight
 import androidx.compose.ui.unit.dp
 import androidx.compose.ui.unit.sp
 import com.bottlerocketstudios.compose.R
 import com.bottlerocketstudios.compose.resources.Colors
 import com.bottlerocketstudios.compose.resources.Dimens
-import com.bottlerocketstudios.compose.resources.greyish_brown
+import com.bottlerocketstudios.compose.resources.typography
 import com.bottlerocketstudios.compose.util.Preview
 import com.bottlerocketstudios.compose.util.PreviewComposable
 import com.bottlerocketstudios.compose.util.ResponsiveText
 import com.bottlerocketstudios.compose.util.asMutableState
+import com.bottlerocketstudios.launchpad.compose.bold
 
 @Composable
 fun PullRequestItemCard(state: PullRequestItemState) {
     Card(
         elevation = Dimens.plane_3,
         modifier = Modifier.wrapContentWidth()
     ) {
-        Row(modifier = Modifier.padding(top = 16.dp, bottom = 16.dp)) {
+        Row(modifier = Modifier.padding(top = Dimens.grid_2, bottom = Dimens.grid_2)) {
             Icon(
                 painter = painterResource(id = R.drawable.ic_repository),
                 contentDescription = stringResource(id = R.string.home_repository_icon),
                 tint = Colors.tertiary,
                 modifier = Modifier
-                    .padding(start = 36.dp, end = 14.dp, top = 6.dp)
+                    .padding(start = Dimens.grid_4_5, end = 14.dp, top = 6.dp)
```

Comment:
Another benefit of grid system is that it highlights unusual padding values, like 14 and 6.  If this doesn't look correct with 16 and 8 dp , then let's add a grid_1_75 and grid_0_75 for 14dp and 6dp respectively.

Reply:
@BottleRocket-Colin ok because I was just about to ask since they aren't multiple is 8 haha